translations/README.zh.md

```diff
@@ -32,11 +32,12 @@ OpenEBS 本身被部署为主机上的另一个容器，支持在每个pod、应
 
 #### *阅读关于这个文档的 [其他语言版本](translations/TRANSLATIONS.md)。*
 
-[🇩🇪](translations/README.de.md)
-[🇷🇺](translations/README.ru.md)
-[🇹🇷](translations/README.tr.md)
-[🇺🇦](translations/README.ua.md)
-[🇫🇷](translations/README.fr.md)
+[🇩🇪](README.de.md)
```

Comment:
This is an interesting find. I wonder if the links in all the pages are broken as they don't have translation prefix. Can you please verify and fix that as part of this PR.

Reply:
Actually, the translations prefixes are the reason why those links were broken. It's because other translations are already in the `translations` directory, so they don't need `translations/README.de.md`.

If they were outside the `translations` directory, for example in the root directory, then the `translations/...` would make sense. The current status is fixed!

You can check links on my forked repo here: https://github.com/aryanrawlani28/openebs/blob/gujarati_language_docs/translations/README.zh.md

And the broken links on current master here:
https://github.com/openebs/openebs/blob/master/translations/README.zh.md

Reply:
Awesome.